src/Controller/SantaController.php

```diff
@@ -119,6 +122,55 @@ public function run(MessageDispatcher $messageDispatcher, Rudolph $rudolph, Requ
         return new Response($content);
     }
 
+    public function sendSampleMessage(Request $request, string $application): Response
```

Comment:
applicationName

Reply:
No, it's the name of the route parameter